src/code/templates/fv-egg-sort-game.js

```diff
@@ -230,9 +230,10 @@ export default class FVEggSortGame extends Component {
     animation: null,
     animatedComponents: [],
     eggs: []
-  }
+  };
 
-  static backgroundClasses = 'fv-layout fv-layout-d'
+  static backgroundClasses = 'fv-layout fv-layout-d';
+  static maxScore = 8;
```

Comment:
Of course this should be based on the number of drakes, but since all spec'd challenges have 8 eggs I figure this is fine for now. I realize I could initialize the variable below in `componentWillMount` based on the size of `drakes`- is that cleaner?